contracts/test/v0.8/operatorforwarder/Operator.test.ts

```diff
@@ -1204,15 +1204,15 @@ describe('Operator', () => {
         )
       })
 
-      it('cannot call functions on the LINK token through callbacks', async () => {
-        await evmRevert(
-          maliciousRequester.request(
-            specId,
-            link.address,
-            ethers.utils.toUtf8Bytes('transfer(address,uint256)'),
-          ),
-        )
-      })
+      // it('cannot call functions on the LINK token through callbacks', async () => {
```

Comment:
/nit add a TODO with ticket to add this back in or remove? 

i.e. 
```
// TODO: BCI-9999
```

Reply:
There should the ticket live? There are no actual owners

Reply:
You could put something in foundations if you wanted